src/snaphu.c

```diff
@@ -155,6 +158,7 @@ void Unwrap(infileT *infiles, outfileT *outfiles, paramT *params,
       MakeTileDir(params,outfiles);
 
       /* different code for parallel or nonparallel operation */
+      #if 0
```

Comment:
This disables the parallel (`fork`) mode. It helps with Windows portability, but we probably still want it on Linux.